docarray/array/mixins/embed.py

```diff
@@ -11,29 +17,40 @@ class EmbedMixin:
     def embed(
         self: 'T',
         embed_model: 'AnyDNN',
+        collate_fn: Optional['CollateFnType'] = None,
```

Comment:
i would move it to the last variable

Reply:
any reason? because you think `collate_fn` should be not be considered as frequently used param atm?